src/connectionbenchmark/connectionbenchmark.cpp

```diff
@@ -40,11 +41,12 @@ void ConnectionBenchmark::initialize() {
 }
 
 void ConnectionBenchmark::setConnectionSpeed() {
-  logger.debug() << "Set speed";
+  logger.debug() << "Set connection speed";
 
-  if (m_bitsPerSec >= Constants::BENCHMARK_THRESHOLD_SPEED_FAST) {
+  // TODO: Take uploadBps for calculating speed into account
```

Comment:
We’ll need to  take the result of the upload benchmark into account when setting the connection speed.